src/main/java/com/internship/backend/controller/LocationController.java

```diff
@@ -31,40 +29,37 @@ public class LocationController {
     @PostMapping("/add")
     public ResponseEntity<Location> addLocation(@RequestBody LocationDTO locationDTO){
         try{
-            Location baseLocation = locationService.fromDTO(locationDTO);
-            baseLocation = locationService.addLocation(baseLocation);
-            Log.info("Added location " + baseLocation.getName());
-            return ok(baseLocation);
-        }catch (LocationAlreadyExistsException e)
-        {
+            Log.info("Added location " + locationDTO.getName());
+            return ok(locationService.addLocation(locationDTO));
+        }catch (LocationAlreadyExistsException e) {
```

Comment:
please ident and format all methods correctly, also valid for other methods. be careful with the missing spaces